### ISSUE TITLE
Persistant Install Fails Causing Havok

### ISSUE THREAD

**Issue by hungrymusic** [DIGImend/digimend.github.io#19]
Ubuntu 20.04 64bit GNOME
Kernel 5.7
Digimend V6
Tablet Ugee M708
AMD GPU rx570

Having read through other similar threads, I am still unable to find either a working or understandable solution. I updated the kernel as recommended and all my headers report back fine.
First time around - DKMS issues abound, resulting in error exit status 10 whenever I tried to do anything. Uninstalled.
After updated kernel, I get the error saying unable to install error 2 (per issue #387).
I have tried this via the .deb and the .tar.gz methods.

No when I boot up, somehow my screen resolution has joined the party. Fortunately this is just about usable at 1024*768 but the monitor is 1920*1080 and no other resolutions are currently accessible. There were some alarming error messages during boot, but they went past too quickly for me to see.

NB I am starting to understand some of the terminal commands but my coding is very much in the copy+paste school.

Thanks in advance x

**Comment by hungrymusic** [comment]
Update
Somehow while trying to remove the incomplete remnants of digimend I managed to uninstall my AMD drivers.
I'm gonna start again from a fresh install :(

**Comment by hungrymusic** [comment]
Right then, fresh install log, in case anyone has the same newbie Linux user issues as me.
Installed Ubuntu 20.04 LTS
Allowed the usual updates (random error locked me out of the session and I had to force restart but nothing else popped up)
Installed Ubuntu Studio Desktop using tasksel (I work in multimedia and the low latency settings are a great starting point for a working Linux studio imho).
Restart
Installed Linux Kernel 5.6 (download and double click the debs in order).
headers...all
headers...lowlatency (in my case, they have specific versions of each file)
modules (as above)
image (as above)
Restart
Installed Vulkan (possibly not relevant but what I did)
Check output/restart
Installed DKMS
Installed git commands/cloning (although didn't use the latter method)
Finally worked out which buttons to press to download the whole kernel drivers folder as a zip.
Extract, open terminal in folder
make dkms install
Restart
Install Krita - BOOM, it's working out of the box.

Sorry for being such an ignorant n00b and royally ####ing my install in the process. I'm now gonna move on to the rest of my setup to include AMD proprietary drivers and Oracle Java (again) - wish me luck and keep doing what you do x
